examples/vision/semisupervised_simclr.py

```diff
@@ -197,14 +207,14 @@ def call(self, images, training=True):
 
 # Image augmentation module
 def get_augmenter(min_area, brightness, jitter):
-    zoom_factor = 1.0 - tf.sqrt(min_area)
+    zoom_factor = 1.0 - np.sqrt(min_area)
     return keras.Sequential(
         [
             keras.Input(shape=(image_size, image_size, image_channels)),
             layers.Rescaling(1 / 255),
             layers.RandomFlip("horizontal"),
             layers.RandomTranslation(zoom_factor / 2, zoom_factor / 2),
-            layers.RandomZoom((-zoom_factor, 0.0), (-zoom_factor, 0.0)),
+            # layers.RandomZoom((-zoom_factor, 0.0), (-zoom_factor, 0.0)),

```

Comment:
Uncomment

Reply:
Yes, thank you. Forgot to push the clean up at first :sweat_smile: